member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.kt

```diff
@@ -305,43 +305,45 @@ class MemberAggregate() {
     }
 
     private fun applyMemberIdentifiedEventFromZignSecSignCommand(cmd: ZignSecSignCommand) {
```

Comment:
Would be interesting if we could make this reusable for different sign options in the future
```suggestion
    private fun maybeApplyMemberIdentified(identity: SignSuccessfullCommand.Identity) {
```